### DISCUSSION TITLE
Error to execute a test

### DISCUSSION THREAD

**Discussion by gustavogmfarias** [prisma/prisma#14562] [Q&A]
Hey guys. It's appearing the follow error to me, but if I execute each test alone, works perfectly.

Even executing in group, as you can see, only the first test doesn't work, the rest works as expected. 

Can anyone help me?  

git: https://github.com/gustavogmfarias/Andrea-Contas-Back-End/blob/1cf34157a782525258246fe249ea854e18db171b/src/modules/accounts/useCases/deleteLojista/tests/DeleteLojistaController.spec.ts#L33

```
 FAIL  src/modules/accounts/useCases/ChangeOwnPassword/tests/ChangeOwnPasswordController.spec.ts

  ● Test suite failed to run

    Command failed: yarn prisma migrate dev
    Error: P1002

    The database server at `localhost`:`8083` was reached but timed out.

    Please try again.

    Please make sure your database server is running at `localhost`:`8083`.

    Context: Timed out trying to acquire a postgres advisory lock (SELECT pg_advisory_lock(72707369)). Elapsed: 10000ms. See https://pris.ly/d/migrate-advisory-locking for details.


error Command failed with exit code 1.

      25 |
      26 |         // Rodar as migrations
    > 27 |         execSync(`yarn prisma migrate dev`);
         |         ^
      28 |         execSync(`yarn seed`);
      29 |     }
      30 |

      at CustomEnvironment.setup (prisma/prisma-environment-jest.js:27:9)


 PASS  src/modules/accounts/useCases/createLojista/tests/CreateLojistaController.spec.ts (69.339 s)
 PASS  src/modules/accounts/useCases/deleteLojista/tests/DeleteLojistaController.spec.ts (72.873 s)  
 PASS  src/modules/accounts/useCases/authenticateLojista/tests/AuthenticateLojistaController.spec.ts (79.461 s)

Test Suites: 1 failed, 3 passed, 4 total
Tests:       8 passed, 8 total
Snapshots:   0 total
Time:        81.999 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by austincrim** [comment]
Hey @gustavogmfarias 👋,

Sorry for the delay in getting to your question. Are you still experiencing testing issues?

**Reply by vladi-stevanovic** [reply]
👋 Hello @gustavogmfarias I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
